.drone.yml

```diff
@@ -28,7 +28,7 @@ steps:
   # test
   - ./scripts/check_gofmt.sh
   - go vet ./...
-  - go test -v ./...
+  - go test --tags unittests -v ./...
```

Comment:
Just checking: this means that `go test ./...` will always try to run e2e tests no? 
In other words, everytime you want to run the unit tests you have to pass that tag. 

If that's true, then it would be more correct to say `go test --tags e2e` instead

Reply:
Good call, if true then defaulting to unit tests and hiding e2e behind a flag would be a better UX.

Reply:
Good point, and I agree. I will change the build tag to be selective for e2e tests, so the default one is for plain unit tests.